### DISCUSSION TITLE
Please fix the control for keyboard player whom used alternate key set

### DISCUSSION THREAD

**Discussion by karnesss** [Drewol/unnamed-sdvx-clone#639] [General]
I'm a 5 years keyboard player and have to play on KSM many times because the alternate key doesn't register when the main key is still being held, this makes a line of notes where on the controller you have to alternate hand to press the same button almost impossible to keyboard player.

https://user-images.githubusercontent.com/112699154/230724477-8dfc05de-1e81-4e8b-bff8-7ed5b0b8ef35.mp4

**Comment by karnesss** [comment]
also please add button that press both fx note like in KSM , i have to work around it using ahk

**Comment by 123jimin** [comment]
It could be because of bounce guard setting, which is set to 10ms by default.

In Settings, go to Input -> Set "bounce guard" setting to 0 for button inputs.

![image](https://github.com/Drewol/unnamed-sdvx-clone/assets/835369/0ebc78ab-0d4d-40ed-b952-c29a03e76ffb)

With this most jacks should be managable.

https://github.com/Drewol/unnamed-sdvx-clone/assets/835369/4a9656d4-a16d-4ada-ac49-b5d3966d9826

I don't think that following the KSM's way is a good idea here as it would give unfair advantages to keyboard players over controller players. Also, Konasute works in the same way USC does; alternative key would not be registered when one key is already pressed for it.
